tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot

```diff
@@ -1,74 +1,86 @@
+    [Documentation]    Runs a jupyter notebook to fill the user PVC over 90% and
+    ...    verifies that alert "User notebook pvc usage above 90%" is fired
+    [Tags]    Tier2    ODS-516
+
+    Fill Up User PVC    ${NOTEBOOK_REPO_URL}    ${TEST_ALERT_PVC90_NOTEBOOK_PATH}
+
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    RHODS-PVC-Usage
+    ...    User notebook pvc usage above 90%
+
+    [Teardown]    Teardown PVC Alert Test
+
+Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
+    [Documentation]    Runs a jupyter notebook to fill the user PVC over 100% and
+    ...    verifies that alert "User notebook pvc usage at 100%" is fired
+    [Tags]    Tier2    ODS-517
+
+    Fill Up User PVC    ${NOTEBOOK_REPO_URL}    ${TEST_ALERT_PVC100_NOTEBOOK_PATH}
+
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    RHODS-PVC-Usage
+    ...    User notebook pvc usage at 100%
+
+    [Teardown]    Teardown PVC Alert Test
+
 
 *** Keywords ***
-Set Up Alert Test
-    [Arguments]  ${NOTEBOOK_PATH}
-    Set Library Search Order  SeleniumLibrary
-    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
-    Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-    Wait for RHODS Dashboard to Load
-    Iterative Image Test  s2i-generic-data-science-notebook  ${notebook_repo_url}  ${NOTEBOOK_PATH}
-    Capture Page Screenshot
-
-Clean Up Files And End Web Test
-    [Documentation]  We delete the notebook files using the new -and expererimental- "Clean Up User Notebook" because "End Web Test" doesn't work well when disk is 100% filled
+Alerts Suite Setup
+    [Documentation]    Test suite configuration
+    Set Library Search Order    SeleniumLibrary
+
+Teardown PVC Alert Test
+    [Documentation]    Deletes user notebook files using the new "Clean Up User Notebook"
+    ...    keyword because "End Web Test" doesn't work well when disk is 100% filled
     Maybe Close Popup
-    Run Keyword And Continue On Failure   Open With JupyterLab Menu   File  Close All Tabs
+    Run Keyword And Continue On Failure    Open With JupyterLab Menu    File    Close All Tabs
     Maybe Close Popup
-    Navigate Home (Root folder) In JupyterLab Sidebar File Browser
-    Delete Folder In User Notebook  ${OCP_ADMIN_USER.USERNAME}  ${TEST_USER.USERNAME}  ods-ci-notebooks-main
-    Sleep  10  reason=Waiting for possible JupyterLab pop-up
+    Navigate Home (Root Folder) In JupyterLab Sidebar File Browser
+    Delete Folder In User Notebook    ${OCP_ADMIN_USER.USERNAME}    ${TEST_USER.USERNAME}    ods-ci-notebooks-main
+    Sleep    10    reason=Waiting for possible JupyterLab pop-up
     Maybe Close Popup
-    Sleep  10  reason=Waiting for possible JupyterLab pop-up
+    Sleep    10    reason=Waiting for possible JupyterLab pop-up
```

Comment:
I think it will be good idea to put "s" for seconds  after the digit

Reply:
Fixed